TAG

```diff
@@ -1 +1 @@
-2.3.108-tb
+2.4.0-rc1
```

Comment:
so if rc1 goes ga, we change that to 2.4.0?

Reply:
I was basing this on what we did for 2.2 in https://github.com/daos-stack/daos/commit/e68757d5534f2c0bab07da1a6415c3ce6475d81a

Reply:
got it